src/providers/soundcloud.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const cheerio = require('cheerio')
+const got = require('got')
+
+module.exports = async username => {
+  const { body } = await got(`https://soundcloud.com/${username}`)
+  const $ = cheerio.load(body, { xmlMode: true })
```

Comment:
why this is necessary? what happens if it is not enabled? 🤔

Reply:
Since I'm querying inside a `<noscript>` tag, per documentation the `xmlMode` is suggested to treat the DOM document rather than HTML.  This is because under normal circumstances the content of `<noscript>` tag is not rendered in browsers (unless JavaScript is disabled, that is).

Despite that Cheerio doesn't parse the page like jsdom or a browser, I found that it helps to treat the document as xml here. It consistent still work without this enabled, but the queries might not be consistent so I used it here, just to be safe.